### DISCUSSION TITLE
Cannot reach CELERY_EMAIL_BACKEND django.core.mail.backends.smtp.EmailBackend

### DISCUSSION THREAD

**Discussion by Neurozone** [mediacms-io/mediacms#267] [Q&A]
**Describe the issue**
MediaCMS cannot send email.
It ends with that error in celery_short.log even with correct settings for smtp (same as server itself)

**To Reproduce**
Steps to reproduce the issue:
Trying to send an email from MediaCMS

**Expected behavior**
email sent

**Screenshots**

**Environment (please complete the following information):**
 - cat /etc/lsb-release
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=20.10
DISTRIB_CODENAME=groovy
DISTRIB_DESCRIPTION="Ubuntu 20.10"

 - Installation method:  single server install
 - Browser, if applicable

**Additional context**

**Comment by mgogoulos** [comment]
Hi @Neurozone , I've converted this to a discussion as I believe it is a configuration issue

Have you set configuration as on https://github.com/mediacms-io/mediacms/blob/main/docs/Configuration.md#set-email-settings
 ? If in doubt, share configuration here (with any sensitive info obfuscated).

**Comment by Neurozone** [comment]
Hello @mgogoulos 
Here a re my congi with obfuscated data
`
# email settings
DEFAULT_FROM_EMAIL = "hecate.***@gmail.com"
EMAIL_HOST_PASSWORD = "****"
EMAIL_HOST_USER = "hecate.***@gmail.com"
EMAIL_USE_TLS = True
SERVER_EMAIL = DEFAULT_FROM_EMAIL
EMAIL_HOST = "smtp.gmail.com"
EMAIL_PORT = 587
ADMIN_EMAIL_LIST = ["hecate.***@gmail.com"]
`

with gmail or my personnal smtp server

**Comment by mgogoulos** [comment]
Did you manage to get through this? 
If no, try to see if email settings are correct, thus we exclude the case of error credentials and have to see whether there is any other issue introduced

Enter the Django shell by running this:

```
source  /home/mediacms.io/bin/activate
python manage.py shell
```

then c/p the following (you might want to change the email to the one you are sending this test email to!)

```
from django.core.mail import EmailMessage
from django.conf import settings

settings.EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'

email = EmailMessage(
    'title',
    'msg',
    settings.DEFAULT_FROM_EMAIL,
    ['mgogoulos@gmail.com'],
)
email.send(fail_silently=False)
```

If this breaks with an error, you are going to see the error - example upon providing error gmail credentials, this gives 

```
SMTPAuthenticationError: (535, b'5.7.8 Username and Password not accepted. Learn more at\n5.7.8  https://support.google.com/mail/?p=BadCredentials d4sm12687785wrc.34 - gsmtp')
```

**Comment by bananacy** [comment]
I fixed it by using this:

EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'

Then i resovled authentication issues with this (might not be an issue with you)
EMAIL_USE_SSL = True
EMAIL_PORT = 465
